examples/stable_fluid.py

```diff
@@ -317,7 +317,7 @@ def reset():
         step(mouse_data)
 
     gui.set_image(dyes_pair.cur)
-    #gui.set_image(np.concatenate(
-    #        [np.abs(velocities_pair.cur.to_numpy() * 0.01 + 0.5),
-    #            np.zeros((res, res, 1))], axis=2))
+    #gui.set_image(velocities_pair.cur.to_numpy() * 0.01 + 0.5)
+    #divergence(velocities_pair.cur); gui.set_image(velocity_divs.to_numpy() * 0.1 + 0.5)
+    #vorticity(velocities_pair.cur); gui.set_image(velocity_curls.to_numpy() * 0.03 + 0.5)
```

Comment:
Are you keeping these for future use or do you expect the user to uncomment these lines? Might be a good idea to add some guidance in the comment. Also, doesn't the standard say that there should be a space after #?

Reply:
Yes, I'm expecting them to uncomment one of these to see the different result, the space is **not left** purposely since it's code instead of human-text.